malaria-ios/ViewControllers/RapidFireGame.swift

```diff
@@ -0,0 +1,28 @@
+import UIKit
+
+class RapidFireGame: Game {
+  
+  var entries: [RapidFireGameEntry] = []
+  static let name = "Rapid Fire"
+  
+  init() {
+    let entry1 = RapidFireGameEntry(question: "Question 1 (answer 2)", answers: ["Answer 1", "Answer 2", "Answer 3"], correctAnswer: 1)
+    let entry2 = RapidFireGameEntry(question: "Question 2 (answer 1)", answers: ["Answer 1", "Answer 2", "Answer 3"], correctAnswer: 0)
+    let entry3 = RapidFireGameEntry(question: "Question 3 (answer 3)", answers: ["Answer 1", "Answer 2", "Answer 3"], correctAnswer: 2)
+    
+    entries.appendContentsOf([entry1, entry2, entry3])
+    
+    super.init(numberOfLevels: entries.count, name: RapidFireGame.name)
+  }
+}
+
+class RapidFireGameEntry {
+  
+  let question: String, answers: [String], correctAnswer: Int
```

Comment:
correctAnswer is a int? What means that variable?

Moreover, each variable to their own line:
```swift
let x
let y
let z
```